.github/ISSUE_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@
 - [ ] Python version: <!-- (e.g. 2.7, 3.3, 3.6) -->
 - [ ] Python architecture: <!-- (e.g. x86, x64) -->
 - [ ] Operating system and version: <!-- (e.g. Windows XP, Windows 7, Windows 8, OSX 11, Red Hat, Ubuntu, Centos 6, Centos 7) -->
-- [ ] OpenDSSDirect.py version number: <!-- (e.g. v0.1.0, v0.2.5, v0.3.0) -->
+- [ ] OpenDSSDirect.py version number: <!-- (e.g. v0.1.0, v0.2.5, v0.3.0) You can use `opendssdirect.Basic.Version()` to get this information --> 
```

Comment:
That's just the engine version though:
https://github.com/dss-extensions/OpenDSSDirect.py/blob/bb20ea7ae3cbcc571bd7507c0df215ddcfd294af/opendssdirect/Basic.py#L67-L69

Shouldn't `opendssdirect.__version__` be more appropriate? 

I think that including `opendssdirect.__version__ ` in `opendssdirect.Basic.Version()` is a good idea to get all info in one place, but we can't address that for old releases.